config/locales/blacklight.en.yml

```diff
@@ -245,3 +245,7 @@ en:
     pagination_compact:
       next: Next &raquo;
       previous: "&laquo; Previous"
+  layouts:
+    catalog_result:
+      aria:
+        sidebar: Sidebar
```

Comment:
Is "Sidebar" the best we can do here? How about "Tools"?  Additionally, once a term is chosen, can you run the i18n tools to fix the translations for the other languages rather than keeping them English?

Reply:
By "fix" do you mean using the google translate backend to pre-fill translations?

Reply:
Yes.

Reply:
Updated to use "Toolbar" and its translated versions.